### PR TITLE
Image export permission error handling

### PR DESCRIPTION
**Pull request by zoran15**
Image Export: 
* inline startup script
* log GCS read permission with "ExportFailed:" prefix so it gets picked up by the calling code
* don't retry on permission error

### CODE REVIEW

File: cli_tools/gce_export/main.go [hunk 1]
```diff
@@ -145,9 +148,13 @@ func (b *bufferedWriter) uploadWorker() {
 			}()
 			if err != nil {
 				fmt.Printf("Failed %v time(s) to upload '%v', error: %v\n", i, in, err)
-				if i > 16 {
-					log.Fatal(err)
+
+				// Don't retry if permission error as it's not recoverable.
+				gAPIErr, isGAPIErr := err.(*googleapi.Error)
+				if i > 16 || (isGAPIErr && gAPIErr.Code == 403 && gcsPermissionErrorRegExp.MatchString(gAPIErr.Message)) {
+					log.Fatalf("ExportFailed: Failed to export disk source to GCS. [Privacy->%v<-Privacy]", err)
```
Comment (adjackura) [inline cli_tools/gce_export/main.go]: Don't log with this style in this tool. This should be wrapped by something else higher up the stack

Comment (zoran15) [reply cli_tools/gce_export/main.go]: How?

**Pull request merged**